attestation-service/verifier/src/snp/mod.rs

```diff
@@ -158,18 +160,27 @@ pub(crate) fn verify_report_signature(
     vendor_certs: &VendorCertificates,
 ) -> Result<()> {
     // check cert chain
-    let VendorCertificates { ask, ark } = vendor_certs;
-    let vcek = verify_cert_chain(cert_chain, ask, ark)?;
+    let VendorCertificates { ask, ark, asvk } = vendor_certs;
+    let vcek = verify_cert_chain(cert_chain, ask, ark, asvk)?;
```

Comment:
This could be a vcek or a vlek, so we might want a different name. As is the check below where we see if it is a vcek is a bit confusing.